### DISCUSSION TITLE
Automatic export in continuous capture

### DISCUSSION THREAD

**Discussion by Totoche95** [bengtmartensson/IrScrutinizer#507] [Ideas]
Thanks for this program, works fine also on 433Mhz with RX470-4 / WL102-341 433 Mhz modules, can let in continuous capture.
Is it possible to export automatically the captured signal, just have a click on Export each time a new signal is captured ?
Just an other question, i tried to increase the 10 seconds start capture timeout , also in Girs : #define DEFAULT_BEGINTIMEOUT 30000UL // milliseconds, but always 10 sec begin timout.
Cheers

**Comment by bengtmartensson** [comment]
> Is it possible to export automatically the captured signal, just have a click on Export each time a new signal is captured ?

Not quite sure what you mean, but on the "Scrutinize signal" pane you can press "Export" during an ongoing continuous capture. There is no way to automate the exports, and it is also not planned. The idea is that if you want to capture a number of signals, you use "Scrutinize remote", either parametrized or raw. and export that as a file when finished.  Please let me know more about your usecase.

>  i tried to increase the 10 seconds start capture timeout , also in Girs : #define DEFAULT_BEGINTIMEOUT 30000UL // milliseconds, but always 10 sec begin timout.

See #273, this is incompletely implemented. But for the start timeouts, it is not really an issue, since the program tries again.

**Comment by Totoche95** [comment]
Thanks, i did not see that. It works fine if i let Scrutinize remote capture in raw mode an Copy selected to Scrutinize signal, i retrieve the signals in the plot to verify the type of signal an then i can export it. It can also can export directly from Scrutinize remote.
I wanted to see my alarm sensors without having to return to my PC each time i open a door or a window.
It works fine for that also.
